index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+        <meta charset="utf-8">
+        <title>Assignment HTML</title>
+
+    </head>
+
+    <body>
+
+        <header id="headline"> 
+
+            <h1> Philipp Aubert </h1> 
+     
+
+            <nav>
+        
+                <ul>                
+                    <li><a href="#">Contact</a></li>
+                    <li><a href="#">Career</a></li>
+                </ul>
+        
+            </nav>
+            
+            <figure> 
+                <img src="IMG_1093.jpg" alt="me"> 
+                <figcaption>Me, 2018. Short hair. Somewhere in Shibuya. Probably not sober. Can't remember though</figcaption>
+            </figure>     
+            
+        </header>
+
+        <main>
+
+        <section id="statement"> 
+
+          <h2>Interests</h2>
+
+          <article> 
+            <h2>Abstract Painting</h2>
+            <p>Challening space with the line</p>
```

Comment:
I think you want to write here `Challenging`?

Reply:
I did. What a blunder :satisfied: